src/Avalonia.Diagnostics/Diagnostics/Screenshots/FileConvetionHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Threading.Tasks;
+using Avalonia.Controls;
+
+namespace Avalonia.Diagnostics.Screenshots
+{
+    /// <summary>
+    /// Take a Screenshot on file by convention
+    /// </summary>
+    public sealed class FileConvetionHandler : BaseRenderToStreamHandler
```

Comment:
Do we need it? This one is easy to implement in the actual application, if developer needs it.